### DISCUSSION TITLE
"Client scopes tab" doesn't exist

### DISCUSSION THREAD

**Discussion by eggyknap** [keycloak/keycloak#13864] [Archive]
The docs [say:](https://www.keycloak.org/docs/latest/server_admin/index.html#_client_scopes)

> Linking between a client scope and a client is configured in the Client Scopes tab of the client.

I don't see a "Client scopes" tab in my clients, nor do I find any mention of scopes under the tabs I do see. What's up?
![Selection_204](https://user-images.githubusercontent.com/59133/185423784-14ae37e6-a903-428d-8d4a-ed750736b151.png)

**Comment by darius-m** [comment]
Some default clients (e.g. the realm clients) have special (internal) configurations, and should not be changed. Create a custom OIDC client and you will be able to see the `Client scopes` tab. As a note, the master realm is meant for administration only.

You should first create a new realm and add your clients in there. Each realm's configuration parameters are isolated (e.g. federated logins, identity providers), so you may choose to create a realm for each organization that you federate authentication for (i.e., each organization provides their own clients and a database of users that may access the clients).

**Reply by eggyknap** [reply]
That makes sense, and now that you've pointed me in the right direction, I *do* see the tab for the custom clients I've looked at. Thanks.
